### PR TITLE
Add initial set of Perl libraries

### PR DESCRIPTION
**Pull request by ferki**
I've collected a highly subjective list of a few Perl libraries that I consider a good set to have initially.

The list was mostly inspired by my personal experience and perception of popular modules from [MetaCPAN leaderboard](https://metacpan.org/favorite/leaderboard), and recommended modules from [Task::Kensho](https://metacpan.org/pod/Task::Kensho). Hopefully it would serve as a good base for others to add more later.

Please review and merge, or let me know how to improve if further!

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -104,6 +110,25 @@
     "image": "https://d3js.org/logo.svg",
     "language": "JavaScript"
   },
+  "Dancer": {
+    "imports": ["Dancer"],
+    "technologies": ["Web Development"],
```
Comment (peti2001) [inline libraries.json:115]: Can MVC be added to?

File: libraries.json [hunk 2]
```diff
@@ -398,6 +441,25 @@
     "image": "https://raw.githubusercontent.com/mockito/mockito.github.io/master/img/logo%402x.png",
     "language": "Java"
   },
+  "Mojolicious": {
+    "imports": ["Mojolicious"],
+    "technologies": ["Web Development"],
```
Comment (peti2001) [inline libraries.json:446]: Same here, is it MVC too?

File: libraries.json [hunk 3]
```diff
@@ -579,6 +653,13 @@
     "description": "An alternative side effect model for Redux apps.",
     "language": "JavaScript"
   },
+  "Rex": {
+    "imports": ["Rex"],
+    "technologies": ["Automation"],
```
Comment (peti2001) [inline libraries.json:658]: Can DevOps be added?

File: libraries.json (hunk 1 above)
Comment (ferki) [reply libraries.json:115]: Yes, I think MVC makes sense too.

File: libraries.json (hunk 2 above)
Comment (ferki) [reply libraries.json:446]: Yup, MVC makes sense here as well.

File: libraries.json (hunk 3 above)
Comment (ferki) [reply libraries.json:658]: Yes, I guess that label would help to orient people.

**Comment by ferki** [comment]
@peti2001: thanks for the feedback! I've added the extra labels you pointed out, and updated the PR.

**Comment by peti2001** [comment]
Thanks again

**Comment by ferki** [comment]
Thanks for merging! :tada: I look forward to the results with the next update of my profile (crossref: just posted to slack that the previous update seems to be stuck somehow).

**Pull request merged**